kedro-airflow/kedro_airflow/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Kedro plugin for running a project with Airflow."""
 
-__version__ = "0.6.0"
+__version__ = "0.6.1"
```

Comment:
Could you revert this change? It'll create a new release and we have a separate release process.

Reply:
Will do!